php/context/class-fieldmanager-context-storable.php

```diff
@@ -52,9 +52,18 @@ protected function save( $data = null ) {
 			$this->save_field( $this->fm, $data, $this->fm->data_id );
 		} else {
 			if ( null === $data ) {
-				$data = isset( $_POST[ $this->fm->name ] ) ? $_POST[ $this->fm->name ] : ''; // WPCS: input var okay. CSRF okay. Sanitization okay.
+				$data = isset( $_POST[ $this->fm->name ] ) ? $_POST[ $this->fm->name ] : array(); // WPCS: input var okay. CSRF okay. Sanitization okay.
+
+				// Multiple select and radio fields with no values chosen are left out of the post request altogether.
+				foreach ( $this->fm->children as $child ) {
```

Comment:
Because `serialize_data` is technically a property on `Fieldmanager_Field`, is it safe to assume that `$this->fm` is going to have a `children` property?